packages/components/src/Disclosure/Disclosure.css

```diff
@@ -0,0 +1,64 @@
+/*
+Re: (.arrowIconWrapper > svg > path)
+This ensure we are spinning just the svg graphic and not the
+conter, which result in an odd square rendering glitch when
+displaying the background color together.
+*/
+.arrowIconWrapper > svg > path {
+  transform-origin: 50% 50% 0;
+  transition: transform var(--timing-base);
+}
+
+.flippedVertical > svg > path {
+  transform: rotate(180deg);
```

Comment:
Can we do a `rotateX` here so the arrow flips, rather than spins?

Reply:
good call, and flip works well and I don't need to drill down into the `> svg > path`